slack-lint/src/main/java/slack/lint/util/UExpressionExtensions.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.lint.util
+
+import com.android.tools.lint.detector.api.UastLintUtils
+import org.jetbrains.uast.UExpression
+import org.jetbrains.uast.UReferenceExpression
+import org.jetbrains.uast.tryResolve
+
+fun UExpression.resolveQualifiedNameOrNull(): String? {
```

Comment:
Add this to an existing util file rather than its own dedicated one. Also please write a short doc

Reply:
👍 Fixed in 130aa73100dffe1956555733eca7a78aa967b6c0